app/docs/dev/installation/docker.md

```diff
@@ -28,6 +28,12 @@ We can run Kuma:
 
 `docker run -p 5681:5681 docker.io/kumahq/kuma-cp:{{ page.latest_version }} run`
 
+{% tip %}
+Running administrative tasks without additional authentication is possible by running the container in a host network.
+To do this add `--network="host"` parameter to the above command.
+Alternatively you can `docker exec` into the container and setup kumactl [authentication](/docs/{{ page.version }}/security/api-server-auth/#admin-user-token).
```

Comment:
There's no `kumactl` in the `kuma-cp` image, right? Or should the user install it?

Reply:
I need to add more info here, we only want to get the token (so point 1) and then setup `kumactl` on the local machine.